ur5_demo_descartes/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8.3)
 project(ur5_demo_descartes)
 
 ## Find catkin macros and libraries
-find_package(catkin REQUIRED descartes_core descartes_moveit pluginlib moveit_core roscpp tf_conversions)
+find_package(catkin REQUIRED descartes_core descartes_moveit pluginlib moveit_core roscpp tf_conversions ur_kinematics)
```

Comment:
I did this locally as well, but I also have to fixup the constants in `ur_kin.h` (related to ros-industrial/universal_robot#217).